src/ws/hooks/item.ts

```diff
@@ -85,10 +85,14 @@ const InvalidateItemOpFeedback = (queryClient: QueryClient) => ({
     queryClient.invalidateQueries(memberKeys.current().recycledItems);
   },
   [FeedBackOperation.VALIDATE]: (itemIds: string[]) => {
-    // todo: invalidate the validation query to refetch the validation status
-    itemIds.map((itemId) =>
-      queryClient.invalidateQueries(itemKeys.single(itemId).validation),
-    );
+    itemIds.forEach((itemId) => {
+      // invalidate the item content to get the public attribute
+      queryClient.invalidateQueries(itemKeys.single(itemId).content);
```

Comment:
Why invalidating the content? does it change anything?

Reply:
Because without that, the item is still in private (missing the public attribute), so the frontend can't display the new publication status. Am I wrong ? @pyphilia